### PR TITLE
Repairs for NETBEANS-2760 and NETBEANS-2761

### PR DESCRIPTION
**Pull request by juneau001**
Updated code to make use of an updated Java EE 8 Maven Archetype.  Changed some minor discrepancies that were found in the sources with respect to Java EE 8 support.

### CODE REVIEW

File: enterprise/maven.j2ee/src/org/netbeans/modules/maven/j2ee/ui/wizard/archetype/J2eeArchetypeFactory.java [hunk 1]
```diff
@@ -90,8 +90,8 @@ public Archetype getAnyArchetypeFor(J2eeModule.Type projectType) {
     private static class AppClientArchetypes extends BaseJ2eeArchetypeProvider {
         @Override
         protected void setUpProjectArchetypes() {
-            addJavaEE8Archetype(Profile.JAVA_EE_8_FULL,"io.github.juneau001", "1.0.0", "webapp-javaee8"); //NOI18N
-            addMojoArchetype(Profile.JAVA_EE_7_FULL, "1.1", "appclient-javaee7"); //NOI18N
+            addJavaEE8Archetype(Profile.JAVA_EE_8_FULL,"io.github.juneau001", "1.2", "webapp-javaee8"); //NOI18N
+            addMojoArchetype(Profile.JAVA_EE_7_FULL, "1.2", "appclient-javaee7"); //NOI18N
```
Comment (jGauravGupta) [inline enterprise/maven.j2ee/src/org/netbeans/modules/maven/j2ee/ui/wizard/archetype/J2eeArchetypeFactory.java]: Is appclient-javaee7 archetype version upgrade required? The latest available version is 1.1.

Comment (juneau001) [reply enterprise/maven.j2ee/src/org/netbeans/modules/maven/j2ee/ui/wizard/archetype/J2eeArchetypeFactory.java]: Thanks for checking.  I have done a basic check on some of the other archetype versions and some of them look incorrect.  For example ejb-javaee7 is coded as version 1.2, and I can only find 1.1 on mvn central.  I will look through and make necessary changes, where needed.

Comment (juneau001) [reply enterprise/maven.j2ee/src/org/netbeans/modules/maven/j2ee/ui/wizard/archetype/J2eeArchetypeFactory.java]: Checked archetype versions against https://repo1.maven.org/maven2/org/codehaus/mojo/archetypes/ and updated accordingly.

**Comment by lkishalmi** [comment]
Please rebase on current master. @neilcsmith-net @geertjanw shall this be marked for 11.1?

**Pull request merged**